src/System.Text.Json/tests/Serialization/Value.WriteTests.cs

```diff
@@ -2,12 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Text.Encodings.Web;
+using Newtonsoft.Json;
 using Xunit;
 
 namespace System.Text.Json.Serialization.Tests
 {
     public static partial class ValueTests
     {
+        [Fact]
+        public static void WriteStringWithRelaxedEscaper()
+        {
+            string inputString = ">><++>>>\">>\\>>&>>>\u6f22\u5B57>>>"; // Non-ASCII text should remain unescaped. \u6f22 = 汉, \u5B57 = 字
+
+            string actual = JsonSerializer.Serialize(inputString, new JsonSerializerOptions { Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping });
+            Assert.Equal(JsonConvert.SerializeObject(inputString), actual);
+            Assert.Equal("\">><++>>>\\\">>\\\\>>&>>>\u6f22\u5B57>>>\"", actual);
```

Comment:
Consider also adding a JsonElement.WriteTo test to prove the scenario (it should work just fine since it's just calling Write methods on the already opened writer)